mkdocs.yml_template

```diff
@@ -258,6 +258,7 @@ nav:
       - "Optimize Bar Cut Sizes": gallery/articles/bar_cutting/index.md
       - "Churn Classification": gallery/articles/churn_classification/index.md
       - "Drift Detection": gallery/articles/drift_detection/index.md
+      - "Supply Chain Monitoring": gallery/articles/supply_chain_monitoring/index.md
```

Comment:
Make sure that this corresponds to the order the article is found in the index pages

Reply:
Then, you have to put this line in the order it is represented when you go to the decision support index page

Reply:
If I'm understanding correctly, that means that the [current docs](https://docs.taipy.io/en/develop/gallery/decision_support/) is "wrong", right? The left nav is not in the same order that it appears on the decision support index page

![image](https://github.com/user-attachments/assets/d3da4e76-a77f-4758-83e6-8837d85b713c)

Reply:
Yep, it is wrong. Check the other sections; they have the correct order. Don't know why it is like this for this one...

Reply:
Sure, go ahead! Thanks!